src/mpi/romio/adio/common/lock.c

```diff
@@ -179,13 +179,16 @@ int ADIOI_Set_lock(FDTYPE fd, int cmd, int type, ADIO_Offset offset, int whence,
 }
 #endif
 
-#if defined(ROMIO_XFS)
 int ADIOI_Set_lock64(FDTYPE fd, int cmd, int type, ADIO_Offset offset,
                      int whence,
 	             ADIO_Offset len) 
 {
     int err, error_code;
+#ifdef _LARGEFILE64_SOURCE
```

Comment:
Is this `#ifdef` necessary? Looking at calls to this function, it seems a 64-bit lock cmd is always passed.